libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1154,11 +1154,6 @@ sampleCount WaveClip::GetSequenceStartSample() const
     return TimeToSamples(mSequenceOffset);
 }
 
-sampleCount WaveClip::GetSequenceEndSample() const
```

Comment:
`GetSequenceStartSample` is now used only in `GetBestBlockSize` and that function exists only to give hints for destructive effects that loop over all samples of a channel, grabbing them in blocks and processing.

So that too may disappear, later.